youtube_dl/extractor/mytaratata.py

```diff
@@ -0,0 +1,72 @@
+# coding: utf-8
+
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+
+
+class MyTaratataIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?mytaratata\.com/taratata/(?P<id>[a-zA-Z0-9_\-/]+)'
+    _TEST = {
+        'url': 'http://mytaratata.com/taratata/519/shaka-ponk-camille-et-julie-berthollet-smells-like-teen-spirit-nirvana',
+        'md5': '99657330eb7dec6d63a329d7f26ec93e',
+        'info_dict': {
+            'id': '7174',
+            'ext': 'mp4',
+            'title': u'TARATATA N°519 - Shaka Ponk / Camille et Julie Berthollet "Smells Like Teen Spirit" (Nirvana)',
+            'uploader': 'Taratata',
+            'description': 'Shaka Ponk / Camille et Julie Berthollet "Smells Like Teen Spirit" (Nirvana)',
+            'thumbnail': 'http://static.mytaratata.com/content/image/5a2562a1a5ee5.jpeg',
+        }
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        title = self._og_search_title(webpage)
+        description = self._og_search_description(webpage)
+
+        formats = []
+
+        video_source_re = re.compile(
+            r'data-source="(?P<url>http://videos.air-productions.cdn.sfr.net'
```

Comment:
More relaxed regex. Don't match exact URL.

Reply:
will do.
Just curious why this rule? I though a more precise url would be safer, ie it will break as soon as something changes, so we are sure not to miss important changes in the website structure.

Reply:
No, completely the opposite. More precise URL imposes more restrictions thus easier to break. If they change storage host or path yours will break while matching by only attribute won't.